<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MacroEditor</class>
 <widget class="QMainWindow" name="MacroEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>779</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Macro Editor</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="MacroIdsLabel" name="macroIdsLabel">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Editing Macro(s): </string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="MacroEventsTable" name="macroEventsTable">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="statusTip">
       <string>Left Click to Select Macro Event   ---   Right Click for Options</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::NoDragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::IgnoreAction</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>1</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Index</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Key String</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Delay</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Duration</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Auto Correct</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Screenshot</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSaveButton"/>
  </widget>
  <action name="actionUndoButton">
   <property name="icon">
    <iconset resource="../../resources/MacroMenuResources.qrc">
     <normaloff>:/new/prefix1/undoIcon.png</normaloff>:/new/prefix1/undoIcon.png</iconset>
   </property>
   <property name="text">
    <string>undoButton</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
   <property name="statusTip">
    <string>Undo: ctrl + z</string>
   </property>
  </action>
  <action name="actionSaveButton">
   <property name="icon">
    <iconset resource="../../resources/MacroMenuResources.qrc">
     <normaloff>:/new/prefix1/saveIcon.png</normaloff>:/new/prefix1/saveIcon.png</iconset>
   </property>
   <property name="text">
    <string>saveButton</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Save: ctrl + s</string>
   </property>
  </action>
  <action name="actionRedoButton">
   <property name="icon">
    <iconset resource="../../resources/MacroMenuResources.qrc">
     <normaloff>:/new/prefix1/redoIcon.png</normaloff>:/new/prefix1/redoIcon.png</iconset>
   </property>
   <property name="text">
    <string>redoButton</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
   <property name="statusTip">
    <string>Redo: ctrl + y</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MacroEventsTable</class>
   <extends>QTableWidget</extends>
   <header location="global">view/macro_editor/MacroEventsTable.h</header>
  </customwidget>
  <customwidget>
   <class>MacroIdsLabel</class>
   <extends>QLabel</extends>
   <header location="global">view/macro_editor/MacroIdsLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/MacroMenuResources.qrc"/>
 </resources>
 <connections/>
</ui>
